actors/market/tests/market_actor_test.rs

```diff
@@ -96,6 +99,22 @@ fn simple_construction() {
     assert_eq!(state_data.last_cron, EPOCH_UNDEFINED);
 }
 
+#[test]
+fn label_cbor() {
+    let mut label = Label::String("i_am_random_string____i_am_random_string____".parse().unwrap());
+    let sv_bz = to_vec(&label)
+        .map_err(|e| ActorError::from(e).wrap("failed to serialize DealProposal"))
+        .unwrap();
+    println!("{:?}", sv_bz);
```

Comment:
manually compared these against the output of Go actors, need to be changed to actual assertions